ui/src/literature/containers/LiteratureSearchContainer.jsx

```diff
@@ -56,43 +58,53 @@ function LiteratureSearch({
       </Col>
       <Col xs={24} lg={17}>
         <LoadingOrChildren loading={loading}>
-          <Row type="flex" align="middle" justify="end">
-            <Col xs={24} lg={12}>
-              <NumberOfResultsContainer namespace={namespace} />
-              {numberOfResults > 0 && (
-                <>
-                  <VerticalDivider />
-                  <CiteAllActionContainer namespace={namespace} />
-                </>
-              )}
-            </Col>
-            <Col xs={12} lg={0}>
-              <ResponsiveView
-                max="md"
-                render={() => (
-                  <DrawerHandle
-                    className="mt2"
-                    handleText="Filter"
-                    drawerTitle="Filter"
-                  >
-                    {renderAggregations()}
-                  </DrawerHandle>
-                )}
-              />
-            </Col>
-            <Col className="tr" span={12}>
-              <SortByContainer namespace={namespace} />
-            </Col>
-          </Row>
-          <Row>
-            <Col span={24}>
-              <ResultsContainer
-                namespace={namespace}
-                renderItem={renderLiteratureItem}
-              />
-              <PaginationContainer namespace={namespace} />
-            </Col>
-          </Row>
+          <EmptyOrChildren
+            data={numberOfResults}
+            description="0 results"
+            extraInfo={
+              <div className="i">
```

Comment:
why `italic`? 

also make sure you use semantic html tags, for this instead of css classes.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em#Usage_notes